test/js/base/config.js

```diff
@@ -64,4 +64,138 @@ describe("Config", function() {
+            assert.strictEqual(initialsConfig.align, "center");
+            assert.strictEqual(initialsConfig.image_rotation, 270);
+            assert.strictEqual(initialsConfig.font_family, "Gold");
+            assert.strictEqual(initialsConfig.font_spacing, -10);
+            assert.strictEqual(initialsConfig.rotation, 0);
+
+            initialsConfig = instance.initialsConfig(config, ["step::personalization"]);
+            assert.strictEqual(initialsConfig.frame, undefined);
+            assert.strictEqual(initialsConfig.align, undefined);
+            assert.strictEqual(initialsConfig.image_rotation, undefined);
+            assert.strictEqual(initialsConfig.font_family, undefined);
+            assert.strictEqual(initialsConfig.font_spacing, undefined);
+            assert.strictEqual(initialsConfig.rotation, undefined);
+            assert.deepEqual(initialsConfig.viewport, [324, 283, 250, 250]);
+
+            initialsConfig = instance.initialsConfig(config, [
+                "custom_alias_1",
+                "step::personalization"
+            ]);
+            assert.strictEqual(initialsConfig.frame, "top");
+            assert.strictEqual(initialsConfig.align, "center");
+            assert.strictEqual(initialsConfig.image_rotation, 270);
+            assert.strictEqual(initialsConfig.font_family, "Gold");
+            assert.strictEqual(initialsConfig.font_spacing, -10);
+            assert.strictEqual(initialsConfig.rotation, 0);
+            assert.deepEqual(initialsConfig.viewport, [349, 308, 200, 200]);
+
+            initialsConfig = instance.initialsConfig(config, ["custom_alias_2"]);
+            assert.strictEqual(initialsConfig.frame, "top");
+            assert.strictEqual(initialsConfig.align, "center");
+            assert.strictEqual(initialsConfig.image_rotation, 270);
+            assert.strictEqual(initialsConfig.font_family, "Silver");
+            assert.strictEqual(initialsConfig.font_spacing, -10);
+            assert.strictEqual(initialsConfig.rotation, 0);
+
+            initialsConfig = instance.initialsConfig(config, [
+                "metal_copper",
+                "metal_silver",
+                "metal_gold"
+            ]);
+            assert.strictEqual(initialsConfig.font_family, "Copper");
+
+            initialsConfig = instance.initialsConfig(config, [
+                "custom_alias_2",
+                "metal_copper",
+                "metal_silver",
+                "metal_gold"
+            ]);
+            assert.strictEqual(initialsConfig.font_family, "Silver");
+        });
+
+        it("should be able to match the config of the remote initials config call", async () => {
+            const instance = await new ripe.Ripe("swear", "vyner", { noBundles: true });
+            await instance.isReady();
+
+            let remote = await instance.getInitialsConfigP();
+            let local = instance.initialsConfig(instance.loadedConfig);

```

Comment:
Can we call these methods with the exact same input?
I think this is nice to enforce that these, given the same input, should return the same output.

I'm suggesting something like:

```suggestion
            const ripeConfig = instance.loadedConfig;

            let remote = await instance.getInitialsConfigP(ripeConfig);
            let local = instance.initialsConfig(ripeConfig);
```

Reply:
No, we the `getInitialsConfigP` doesn't work like that